### ISSUE TITLE
bug related to choosing whisper language 

### ISSUE THREAD

**Issue by ixff** [SubtitleEdit/subtitleedit#9127]
1. Cantonese is unavailable if applied language filter in choosing language drop-down menu. This may be due to the difference between language filter list and whisper supported language list.
2. Scroll bar is missing in language drop-down menu if applied language filter and then remove filter, unless reopen the window.

![SubtitleEdit_HltkFYmIY](https://github.com/user-attachments/assets/79731c54-4ffc-4879-93da-8ab994b1575f)

**Comment by Purfview** [comment]
Cantonese  is supported only large-v3 and turbo models.

**Comment by Purfview** [comment]
Yes, `.en` models support only English but you can select all and 'turbo` model doesn't support translation.
